modules/core/src/test/java/org/apache/ignite/internal/GridNodeMetricsLogSelfTest.java

```diff
@@ -58,7 +71,12 @@
         cfg.setExecutorConfiguration(new ExecutorConfiguration(CUSTOM_EXECUTOR_0),
             new ExecutorConfiguration(CUSTOM_EXECUTOR_1));
 
-        cfg.setGridLogger(strLog);
+        cfg.setCommunicationSpi(new TestRecordingCommunicationSpi());
+
+        if (igniteInstanceName.endsWith("0") && crdLog != null)
```

Comment:
For avid of mistake in _GridNodeMetricsLogSelfTest0_ and _GridNodeMetricsLogSelfTest10_, batter to use _GridAbstractTest#getTestIgniteInstanceName(int)_.
This expression can check in assert.